src/nvim/buffer.c

```diff
@@ -2062,16 +2063,17 @@ void buflist_setfpos(buf_T *const buf, win_T *const win,
 }
 
 
-/*
- * Return true when "wip" has 'diff' set and the diff is only for another tab
- * page.  That's because a diff is local to a tab page.
- */
+/// Check that "wip" has 'diff' set and the diff is only for another tab page.
+/// That's because a diff is local to a tab page.
+///
+/// @param  wip  window information
```

Comment:
@justinmk I removed the docstrings since they are already self documented. Please let me know if you want them and I need to find an interesting docstring in these cases.